app/src/main/java/com/example/spacex_launchinfoapp/MainActivity.kt

```diff
@@ -2,10 +2,68 @@ package com.example.spacex_launchinfoapp
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.MenuItem
+import androidx.appcompat.app.ActionBarDrawerToggle
+import androidx.appcompat.widget.Toolbar
+import androidx.core.view.GravityCompat
+import androidx.drawerlayout.widget.DrawerLayout
+import com.google.android.material.navigation.NavigationView
+
+class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {
+
+    private lateinit var drawerLayout: DrawerLayout
+    private lateinit var navigationView: NavigationView
 
-class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
please separate this method to different methods. This method is too long with a different purposes. 
You can do smth like this: 

```
override fun onCreate(savedInstanceState: Bundle?) {
       setContentView(R.layout.activity_main)
       setSupportActionBar(toolbar)
       initDrawer()
       initNavigation()
}
```